recipes/python-funcsigs/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
+
+package:
+  name: python-{{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Super tired of dinking around with this in every package so this is what I'm doing from now on.